src/api.py

```diff
@@ -0,0 +1,18 @@
+import requests 
+import hashlib
+def fetch(url):
+    con = requests.head(url)
+    r = requests.get(url)
+    name = url.split('/')[-1]
+    filename = '../download/'+name
+    with open(filename,'wb') as f:
+        f.write(r.content)
+    response = {
```

Comment:
since this is mapping you could instead return this way:
```
response = dict(
    filename=filename,
    content_type=con.headers['content-type'],
...
```

That said, it could be even better to return an actual object instead.

Reply:
Okay I will be making an object for it